Sources/Noora/Noora.swift

```diff
@@ -33,12 +33,46 @@ public protocol Noorable {
     ///   - collapseOnSelection: When true, the question is collapsed after the question is entered.
     /// - Returns: The option selected by the user.
     func yesOrNoChoicePrompt(
-        title: String?,
-        question: String,
+        title: TerminalText?,
+        question: TerminalText,
         defaultAnswer: Bool,
-        description: String?,
+        description: TerminalText?,
         collapseOnSelection: Bool
     ) -> Bool
+
+    /// It shows a success alert.
+    /// - Parameters:
+    ///   - message: The success message
+    ///   - next: A list of steps that the person could take after.
+    func success(_ message: TerminalText)
+
+    /// It shows a success alert.
+    /// - Parameters:
+    ///   - message: The success message
+    ///   - next: A list of steps that the person could take after.
```

Comment:
I wonder if we should rename `next` to `nextSteps` to make it more obvious what it's for without having to read the documentation for the parameter. Don't feel strongly about it, though.